addons/csw/functions/fnc_positionTripod.sqf

```diff
@@ -0,0 +1,34 @@
+/*
+ * Author: TCVM
+ * Moves the tripod around with the player
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Tripod <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, tripod] call ace_csw_fnc_positionTripod
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params["_player", "_tripod"];
+
+[_player select 1, "forceWalk", "ACE_csw", true] call EFUNC(common,statusEffect_set);
+_tripod attachTo [_player select 1, [0, 2, 0.5]];
```

Comment:
There is some overlap with ace_dragging.
We could use that instead, but it's also more restrictive and wouldn't allow moving while prone.

If we keep our own method, we'll need to add some more scripts to handle player dying, unconscious, getting in vehicles and something to claim the tripod.